knapsack/Python/knapsack.py

```diff
@@ -0,0 +1,37 @@
+def knapsack(v, w, n, W):
+    """
+    Function solves 0/1 Knapsack problem
+    Function to calculate frequency of items to be added to knapsack to maximise value
+    such that total weight is less than or equal to W
+    :param v: set of values of n objects,
+    :param w: set of weights of n objects,
+    first element in v, w is useless
+    relevant values start from index 1
+    :param n: number of items
+    :pram W: maximum weight allowd by knapsack
+    """
+    m = [[0 for j in range(W+1)] for i in range(n+1)]
+    for j in range(W+1):
+        m[0][j] = 0
+
+    for i in range(1, n+1):
+        for j in range(W+1):
+            if w[i] > j:
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpuvqiarge/knapsack/Python/knapsack.py
+++ b/tmp/tmpuvqiarge/knapsack/Python/knapsack.py
@@ -30,7 +30,7 @@
     n = 5
     W = 8
     print("Maximum value possible in given weight limit: ",
-        str(knapsack(v, w, n, W)))
+          str(knapsack(v, w, n, W)))
 
 
 if __name__ == '__main__':
```